pom.xml

```
@@ -304,41 +259,116 @@
     <dependency>
       <groupId>org.postgresql</groupId>
       <artifactId>postgresql</artifactId>
-      <version>9.4.1208</version>
       <scope>test</scope>
     </dependency>
     <dependency>
       <groupId>org.jenkins-ci.plugins</groupId>
       <artifactId>matrix-auth</artifactId>
-      <version>2.3</version>
       <scope>test</scope>
     </dependency>
     <dependency>
       <groupId>org.jenkins-ci.plugins.workflow</groupId>
       <artifactId>workflow-cps</artifactId>
-      <version>2.74</version>
       <scope>test</scope>
     </dependency>
     <dependency>
       <groupId>org.jenkins-ci.plugins.workflow</groupId>
       <artifactId>workflow-basic-steps</artifactId>
-      <version>2.2</version>
       <scope>test</scope>
     </dependency>
-    <!-- parent pom test dependencies needs overriding-->
 
   </dependencies>
 
   <dependencyManagement>
     <dependencies>
+
+
+      <dependency>
+        <!-- Pick up common dependencies for the selected LTS line: https://github.com/jenkinsci/bom#usage -->
+        <groupId>io.jenkins.tools.bom</groupId>
+        <artifactId>bom-2.277.x</artifactId>
```

Shouldn't this match the jenkins.version value taht was assigned in the properties section?
```suggestion
        <artifactId>bom-2.222.x</artifactId>
```

---

Both options are possible, will commit fix to use 222.x bom, to do not confuse any body